Cargo.toml

```diff
@@ -20,6 +20,7 @@ std = []
 
 [build-dependencies]
 parse-zoneinfo = { version = "0.3" }
+regex = { default-features = false, version = "1"  }
```

Comment:
Does it work to make this optional, and put it behind a `"filter-timezones"` feature? I'm genuinely not sure if you can `cfg(feature = "filter-timezones")` in a build.rs file.

Reply:
I can't swear to it, but I don't think it's possible. I tried making a commit and it is easy enough to refactor `chrono-tz` itself to use the feature and conditionally compile out all the relevant regex code, but then using it from the test example doesn't seem to work. I think features are not passed through as one would want. This might be relevant:https://github.com/rust-lang/cargo/issues/5499